docker-compose.yml

```diff
@@ -33,16 +37,8 @@ services:
     build: ./.docker/adminer
     restart: always
     ports:
-      - ${ADMINER_PORT}:8080
-
-  util:
-    image: swa-util:0001
-    build:
-      context: ./.docker/util
-      dockerfile: Dockerfile
-    volumes:
-      - ./.docker/www:/var/www/html
-      - ./:/swa
+      - ${ADMINER_PORT:-5556}:8080
 
 volumes:
   db-volume:
+  joomla-logs:
```

Comment:
Should this be a mounted volume instead for easier access to the logs?

Reply:
Could be, the reason it is not just using the dir in www though is because of permissions issues.
as the docker user (or user being used in the docker container) can't write to www on the host system by default.

So the best way to see logs would probably be to look inside the dir in the container, or perhaps they even get output to stdout?